packages/core-utils/src/app/misc.ts

```diff
@@ -190,6 +190,20 @@ export const hexStrToNumber = (hexString: string): number => {
   return parseInt(remove0x(hexString), 16)
 }
 
+/**
+ * Parses a hex string if one is given otherwise returns the number that was
+ * given
+ * @param str the String to parse
+ * @returns the parsed number.
+ */
+export const maybeParseHexString = (
```

Comment:
As much as I like the `maybe`, `returnInputAsNumber` might be a better name 😄

Reply:
Oh haha I was going with `maybe` as in the [maybe type](https://en.wikipedia.org/wiki/Option_type) although there maybe refers to a noun not a verb. In any case `castToNumber` is more clear: Fixed! -> [Rename maybeParseHexString to castToNumber](https://github.com/ethereum-optimism/optimism-monorepo/pull/36/commits/181476e330dba226aaeab306d1102142da0d05ed)